### DISCUSSION TITLE
Não consigo iniciar (start) a impressora

### DISCUSSION THREAD

**Discussion by priraraujjo** [porrey/Virtual-ZPL-Printer#47] [General]
Ao clicar no botão "Start" não inicia e da uma mensagem "Foi feita uma tentativa de acesso a um soquete de uma maneira..", não consigo visualizar toda mensagem. Se puderem ajudar nisso. Grata

**Comment by porrey** [comment]
Possible issues:

1. The selected IP address is either invalid
2. The port is being used already
3. You do not have access (admin privileges) to the IP and port.
